api/src/services/settings.ts

```diff
@@ -1,8 +1,24 @@
-import type { AbstractServiceOptions } from '@directus/types';
+import type { AbstractServiceOptions, OwnerInformation } from '@directus/types';
 import { ItemsService } from './items.js';
+import { sendReport } from '../telemetry/index.js';
 
 export class SettingsService extends ItemsService {
 	constructor(options: AbstractServiceOptions) {
 		super('directus_settings', options);
 	}
+
+	async setOwner(data: OwnerInformation) {
+		const { project_id } = await this.knex.select('project_id').from('directus_settings').first();
+
+		sendReport({ ...data, project_id }).catch(async () => {
+			await this.knex.update('project_status', 'pending').from('directus_settings');
+		});
+
+		return await this.upsertSingleton({
+			project_owner: data.email,
+			project_usage: data.project_usage,
+			org_name: data.org_name,
+			product_updates: data.product_updates,
```

Comment:
Should this not also be updating the project status?
```suggestion
			product_status: null
```